File: MinMaxPairSum.js
var minPairSum = function(nums) {
    var sums = 0;
    nums.sort((a, b)=>a-b);
    for (let index = 0; index < nums.length / 2; index++){
        sums = Math.max(sums, (nums[index] + nums[nums.length - 1 - index]));
    }
    return sums;
    
};

File: MinMaxPairSum.py
class Solution:
    def minPairSum(self, nums: List[int]) -> int:
        sums = 0
        nums.sort()
        for index in range(len(nums) // 2):
            sums = max(sums, (nums[index] + nums[len(nums) - index - 1]))
        return sums
